provider-ui/package-lock.json

```diff
@@ -7103,9 +7103,9 @@
       },
```

Comment:
I've pushed this because I noticed the `package-lock.json` in provider-ui has the updated version of the `csstype` package(3.0.4), I think it's better if we keep it that way, makes both lock files consistent?

Reply:
The updates for the deps are handled by the Dependabot and need not be handled by us. That's the reason I asked you to remove it.

Reply:
Okay. Is it fine if I make a new commit by reverting the changes to the original?

Reply:
You can do anything on this PR no issues, until and unless it doesn't hamper anything else😅.